tools/test/test262/excludelist.orig.xml

```diff
@@ -37,11 +37,288 @@
     <test id="annexB/built-ins/RegExp/prototype/compile/this-subclass-instance"><reason>TODO</reason></test>
     <test id="annexB/language/comments/multi-line-html-close"><reason>TODO</reason></test>
     <test id="annexB/language/function-code/block-decl-func-skip-arguments"><reason>TODO</reason></test>
+    <test id="annexB/language/function-code/if-decl-else-decl-a-func-skip-early-err-switch"><reason>TODO</reason></test>
```

Comment:
I believe there was an unexpected error while generating excludelist

Reply:
The `make_excludelist.py` doesn't gave me any error.